QrngWebservice/QrngWebserviceStartup.cs

```diff
@@ -43,10 +43,12 @@ public void Execute()
             QrngWebserviceSettings settings = UserSettings.GetSettings(typeof(QrngWebserviceSettings)) as QrngWebserviceSettings;
             if (string.IsNullOrEmpty(settings.Username) || string.IsNullOrEmpty(settings.Password.Value))
             {
-                MessageBox.Show("The credentials for accessing the webservice that retrieves the random numbers are empty." +
+                //MessageBox.Show(
+                throw new ArgumentNullException(
+                    "The credentials for accessing the webservice that retrieves the random numbers are empty." +
                     "\n\rIf you don't have the credentials you can subscribe freely at https://qrng.physik.hu-berlin.de" +
-                    "\n\r\n\rYou can enter or change your credentials at any time in Fairmat Settings->Plugins settings",
-                    "QRNG");
+                    "\n\r\n\rYou can enter or change your credentials at any time in Fairmat Settings->Plugins settings");
+                //, "QRNG");
```

Comment:
Ho controllato su desktop, il codice che gestisce l'inizializzazione del form esegue tutte le operazioni in un try catch ed eventualmente logga su console l'eccezione senza rethrow.

Quindi di fatto fare direttamente un MessageBox.Show dovrebbe essere totalmente superfluo, e il codice non causerebbe crash in questo modo.